### DISCUSSION TITLE
Free SQL IDE Support?

### DISCUSSION THREAD

**Discussion by ryanhamilton** [crate/crate#16121] [Q&A] [answered]
Hi,

QStudio is a free light-weight SQL IDE that is trying to support almost every database:
https://www.timestored.com/qstudio/database/

I assume your old crateDB driver is long gone and the recommendation is to use postgresql?
If so I will start hiding it in our UI.

Additionally, would anyone be interested in testing qStudio + crate?
In particular we normally see issues around data types in results and the server tree listing.
If you find any issues, either post here or on our github:
https://github.com/timeseries/qstudio/issues

Thanks.

![image](https://github.com/crate/crate/assets/7273208/1320525c-e9c4-49ee-8184-c0d606a26191)

**Comment by matkuliak** [comment, marked as answer]
Hi @ryanhamilton,

Thank you for reaching out. I played around with qStudio with both our cloud and local versions.
I got it working with postgresql, you're correct. I haven't tested extensively but it worked as it should in my experience. If we go ahead with more extensive testing and see any issues, we will let you know. 

Thank you!
